buildSrc/src/main/kotlin/korlibs/korge/gradle/util/HttpServerTools.kt

```diff
@@ -175,6 +175,7 @@ fun File.miniMimeType() = when (this.extension.toLowerCase()) {
     "svg" -> "image/svg+xml"
     "mp3" -> "audio/mpeg"
     "wasm" -> "application/wasm"
+    "mjs" -> "text/javascript"
```

Comment:
Maybe `”js”, “mjs” ->`?

Reply:
`js` is correctly recognized by `Files.probeContentType`. But I can add it for completeness' sake.

Reply:
Yes, add it too. In the end we cannot control directly which files are detected. Probably PNG and HTML are already detected but are there also. Maybe newer JVM versions are already providing the right mime-types. But since we are providing others too, let's provide all the ones that we might want to serve so this is complete.

Reply:
Done👌